ast/ddl.go

```diff
@@ -369,7 +369,64 @@ type ColumnOption struct {
 
 // Restore implements Node interface.
 func (n *ColumnOption) Restore(ctx *RestoreCtx) error {
-	return errors.New("Not implemented")
+	switch n.Tp {
+	case ColumnOptionNoOption:
+		return nil
+	case ColumnOptionPrimaryKey:
+		ctx.WriteKeyWord("PRIMARY KEY")
+	case ColumnOptionNotNull:
+		ctx.WriteKeyWord("NOT NULL")
+	case ColumnOptionAutoIncrement:
+		ctx.WriteKeyWord("AUTO_INCREMENT")
+	case ColumnOptionDefaultValue:
+		ctx.WriteKeyWord("DEFAULT")
+		if n.Expr != nil {
+			ctx.WritePlain(" ")
+			if err := n.Expr.Restore(ctx); err != nil {
+				return errors.Annotate(err, "An error occurred while splicing ColumnOption DefaultValue Expr")
+			}
+		}
+	case ColumnOptionUniqKey:
+		ctx.WriteKeyWord("UNIQUE KEY")
+	case ColumnOptionNull:
+		ctx.WriteKeyWord("NULL")
+	case ColumnOptionOnUpdate:
+		//todo Waiting for FuncCallExpr
+		//ctx.WriteKeyWord("ON UPDATE")
+		//if n.Expr != nil {
+		//	ctx.WritePlain(" ")
+		//	if err := n.Expr.Restore(ctx); err != nil {
+		//		return errors.Annotate(err, "An error occurred while splicing ColumnOption ON UPDATE Expr")
+		//	}
+		//}
+	case ColumnOptionFulltext: //There is no parsing in goyacc
```

Comment:
```suggestion
 	case ColumnOptionFulltext: 
 	// TiDB Parser ignore the `ColumnOptionFulltext` type now
```
and i think we should throw a error here.